### ISSUE TITLE
Review progress

### ISSUE THREAD

**Issue by marisid** [CYPIAPT-LNDSE/social-up#39]
Hi @MatteoTurco, we have deployed a prototype online without the Facebook page with the Chrome extension. You can find it here: https://cypiapt-lndse.github.io/social-up/

We have created a mock extension, which we will presenting this afternoon but it only lives locally at the moment. So, imagine that the landing page you see first will be appearing once you click on an icon in the top bar of your Facebook account.

We are still working on some aspects of the pages - including an animation for the road map with the exercises. We'll keep you updated once we have it online. 

In the meantime, could you please have a look and let us know your thoughts and feeback. We have time for some changes this afternoon. Thanks!

**Comment by MatteoTurco** [comment]
Looks fine to me. I can only recommend adding "ashamed" to the list of feelings, as this is q common emotional reaction amongst teenagers experiencing low self-esteem. But it is not essential to add this as the list looks very comprehensive.

**Comment by marisid** [comment]
Yes, no problem @MatteoTurco. We could add *ashamed* in the list too. 

Just wanna check with you if you went through the whole user journey and whether it was clear to you. 

Once the user fills in their mood and strength of feelings on the landing page, different options appear and then the user can check the __facts__ page, the __exercises__ page or the __testimonials__ page. 

How did it feel as a user? By the way, we have the animation working on the __exercises__ page now but will put it online at around 2:30.
